src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -790,9 +790,9 @@ def aks_create(cmd,
                edge_zone=None,
                enable_secret_rotation=False,
                disk_driver_version=None,
-               disable_disk_driver=None,
-               disable_file_driver=None,
-               disable_snapshot_controller=None,
+               disable_disk_driver=False,
```

Comment:
what if `disable_disk_driver` not specified, it would be false, that would change the default behavior

Reply:
o, default `disable_disk_driver` value should be false

Reply:
When it is false, it wouldn't change the default behavior. Because `disable_disk_driver=False` means the customer doesn't specify the value. Previously, it meant customers want to disable the disk CSI driver.

In the implemented logic,  it is compatible currently.